src/Native/Graphics/Element.js

```diff
@@ -334,6 +334,7 @@ Elm.Native.Graphics.Element.make = function(localRuntime) {
 
 			case 'Z':
 				transform = 'translateX(' + ((-w/2)|0) + 'px) ';
+				break;
```

Comment:
These actually are supposed to fall through. Only excuse I can make is that this was written by a younger me ;P